app/models/user.rb

```diff
@@ -7,6 +7,8 @@ class User < ApplicationRecord
   validates :date_of_birth, presence: true
   validates :email, format: { with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i, on: :create }
 
+  enum role: [:admin, :member, :banned]
```

Comment:
dangerous. what if in the future we will need add a role like `premium_member` ? it would make sense to have it between `admin` and `member` but if we do that we will break the existing application.